dts/xtensa/espressif/esp32s2.dtsi

```diff
@@ -25,6 +25,11 @@
 		};
 	};
 
+	wifi: wifi {
+		compatible = "espressif,esp32-wifi";
+		status = "disabled";
+	};
+
```

Comment:
What is this supposed to do? There's no binding for this compatible.

Reply:
@mbolivar-nordic I think we can remove all entries regarding this in all Espressif SoCs. Current wifi implementation does not use this binding at all. I'll submit a PR, sounds good removing this whole entry?

Reply:
Sure